site_root/input/resources/examples/Patient-pt-001.json

```diff
@@ -1,15 +1,54 @@
 {
-    "resourceType":"Patient",
-    "id": "pt-001",
-    "meta": {
-        "profile": [
-            "http://fhir.kids-first.io/StructureDefinition/participant"
-        ]
+  "resourceType": "Patient",
+  "id": "pt-001",
+  "meta": {
+    "profile": [
+      "http://fhir.kids-first.io/StructureDefinition/participant"
+    ]
+  },
+  "extension": [
+    {
+      "url": "http://hl7.org/fhir/us/core/StructureDefinition/us-core-ethnicity",
+      "extension": [
+        {
+          "url": "ombCategory",
```

Comment:
Interesting 🤔 didn't know this would be a valid URI

Reply:
Yeah! So, if we look at the [extensions](https://github.com/kids-first/kf-model-fhir/blob/initial-participant-conf-resources/site_root/input/resources/extensions/StructureDefinition-us-core-ethnicity.json), these extensions have two nested extensions, `"ombCategory"` and `"detailed"`, and their `"fixedUri"`s are `"ombCategory"` and `"detailed"`.

Reply:
> Interesting 🤔 didn't know this would be a valid URI

It isn't.